### ISSUE TITLE
enc_input

### ISSUE THREAD

**Issue by haoyangz** [wiseodd/controlled-text-generation#6]
[This](https://github.com/wiseodd/controlled-text-generation/blob/master/ctextgen/model.py#L222) line seems to assume that the text between sos and eos is always long enough such that there is no padding at the end of the sentence. Why?

**Comment by wiseodd** [comment]
Thanks for the comment.
It would probably better to just do this:
```
original_sentence = <start> i want to fly <eos>
enc_input = <start> i want to fly <eos> = original_sentence
dec_input = <start> i want to fly <eos> = original_sentence
dec_target = i want to fly <eos> <pad>
```

**Comment by wiseodd** [comment]
Fixed in 6f763ad
